tests/unit/auth/v2/errorHandling.js

```diff
@@ -26,10 +28,13 @@ describe('Error handling in checkAuth', () => {
             url: '/bucket',
             query: {},
         };
+        const requestContext = new RequestContext(request.headers,
```

Comment:
I think we can do some cleanup of code duplication while we are here. It would be ideal to move this code into a check function and just pass only the required stuff.